integration-tests/src/test/resources/performance-features/continuously-create-bridges.feature

```diff
@@ -0,0 +1,130 @@
+Feature: Continuous create Bridges performance tests
+
+  @continuously-create-bridges
+  Scenario Outline: Continuously create Bridges by <users> users
+    When create benchmark with content:
+      """text/vnd.yaml
+      name: rhose-create-bridges
```

Comment:
This hyperfoil name benchmark may clash with this one https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/resources/performance-features/create-bridges-at-once.feature#L10 in case running parallel perf test... which may not happen, but just in case.

Reply:
Good point, will rename